tests/unit/test_mnist_csv_format.py

```diff
@@ -266,4 +266,4 @@ def test_can_import(self):
     @mark_requirement(Requirements.DATUM_GENERAL_REQ)
     def test_can_detect(self):
         detected_formats = Environment().detect_dataset(DUMMY_DATASET_DIR)
-        self.assertEqual([MnistCsvImporter.NAME], detected_formats)
+        self.assertIn(MnistCsvImporter.NAME, detected_formats)
```

Comment:
Please fix `MnistCsvImporter` to detect a csv file which has the filename pattern "mnist_*.csv".

Reply:
if there's ".csv" file in the folder, "tabular" will be always detected. 
So this can't be reverted. But in the test_tabular_format.py, "mnist_csv" can't be detected and we can compare them using "equality".

Reply:
We have `FormatDetectionConfidence` mechanism. Give the higher confidence to `MnistCsv` in those case.